scale-encode/src/impls/primitive_types.rs

```diff
@@ -14,7 +14,8 @@
 // limitations under the License.
 
 use crate::{error::Error, EncodeAsType};
-use primitive_types::{H128, H160, H256, H384, H512, H768};
+use alloc::vec::Vec;
+pub use primitive_types::{H128, H160, H256, H384, H512, H768};
```

Comment:
Changed this to pub use, because if primitive types are used, these types are needed as well I believe.

Reply:
These aren't exported from the mod, so it's a bit pointless at the mo :)

I think better anyway would be to just export the whole of `primitive_types` from `lib.rs` if the feature is enabled; in other crates I've done the following so for consistency maybe add this to `lib.rs`:

```rust
pub mod ext {
    #[cfg(feature = "primitive-types")]
    pub use primitive_types;
}
```